docs/_src/api/api/crawler.md

```diff
@@ -1,53 +1,95 @@
-<a name="entity"></a>
-# Module entity
+<a name="crawler"></a>
+# Module crawler
 
-<a name="entity.EntityExtractor"></a>
-## EntityExtractor Objects
+<a name="crawler.Crawler"></a>
+## Crawler Objects
```

Comment:
Not sure why we always have "Objects" appended to every section title. I think it makes the navigation bar on the right quite cluttered. Is there any section where it's **not** objects? Being very precise it would also be rather classes than objects, I guess.

Reply:
Good point, I removed Objects now.